### PR TITLE
Add tuition link class

### LINKED ISSUE #79: Add student and session index to SessionListPanel
Include student and session index for users to easily locate the index of sessions belonging to a student for a more convenient `delete_session` command usage.

### PR DESCRIPTION
**Pull request by chwoozy**
Add tuition class to link student and sessions for displaying session information with student in UI.

Closes #79. 

Todo:
Tuition testcases.

### CODE REVIEW

**Comment by chwoozy** [comment]
For quick reference, here's an image of the current UI:
image

File: src/main/java/seedu/address/model/tuition/Tuition.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    @Override
+    public int hashCode() {
+        // use this method for custom fields hashing instead of implementing your own
+        return Objects.hash(student, session);
+    }
+
+    @Override
+    public String toString() {
+        final StringBuilder builder = new StringBuilder();
+        builder.append(student.getName())
```
Comment (nowknowing) [inline src/main/java/seedu/address/model/tuition/Tuition.java]: Should we use the studentIndex-Session index here too?
Actually there's no use  of this toString() method currently, right

**Review by nowknowing — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/model/tuition/Tuition.java (hunk 1 above)
Comment (chwoozy) [reply src/main/java/seedu/address/model/tuition/Tuition.java]: Good point 😊 Will make changes!

**Pull request merged**